README.md

```diff
@@ -11,28 +11,19 @@
 
 # ember-frost-sidebar
 
- * [Installation](#Installation)
- * [API](#API)
- * [Examples](#Examples)
- * [Contributing](#Contributing)
+ * [Installation](#installation)
+ * [Components](#components)
+ * [Development](#development)
+ * [Testing](#testing)
 
 ## Installation
 ```
 ember install ember-frost-sidebar
 ```
+## Components
+The following components are available when you install ember-frost-sidebar
 
-## API
-
-| Attribute | Type | Value | Description |
-| --------- | ---- | ----- | ----------- |
-| ` ` | ` ` | ` ` | Coming soon |
-
-## Examples
-
-### Example
-```handlebars
-Coming soon
-```
+* [frost-sidebar](frost-sidebar.md)
```

Comment:
APIs need to be listed here.  We only create a sub page in case when there are more then one components that are part of the repo.  This was a case with frost-core.